artman/cli/main2.py

```diff
@@ -54,7 +54,7 @@ def main(*args):
     # Get to a normalized set of arguments.
     flags = parse_args(*args)
     user_config = read_user_config(flags)
-    _adjust_input_dir(flags.input_dir)
+    _adjust_root_dir(flags.root_dir)
```

Comment:
Is this a no-op `if not flags.root_dir`?

Reply:
There should always have one after normalization. Updated line 290 to normalize this flag.

Reply:
I thought so. Checking again.

Reply:
Confirmed what @ethanbao just said.